lite/proxy/verifier.go

```diff
@@ -8,12 +8,12 @@ import (
 	lclient "github.com/tendermint/tendermint/lite/client"
 )
 
-func NewVerifier(chainID, rootDir string, client lclient.SignStatusClient, logger log.Logger) (*lite.DynamicVerifier, error) {
+func NewVerifier(chainID, rootDir string, client lclient.SignStatusClient, logger log.Logger, cacheSize int) (*lite.DynamicVerifier, error) {
```

Comment:
This signature is getting quite large, would benefit from [functional options](https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis), an example from another part of the codebase:

https://github.com/tendermint/tendermint/blob/0c9c3292c918617624f6f3fbcd95eceade18bcd5/privval/socket.go#L41-L54

Reply:
@xla 
I think function options may be not suitable for `NewVerifier`. Because `NewVerifier` use these parameters for processing network operation, not just variable assignment.